ToggleButton/ToggleButton.js

```diff
@@ -160,6 +169,7 @@ const ToggleButtonBase = kind({
 				{...props}
 				aria-pressed={props.selected}
 				css={css}
+				decoration={props.skin === 'silicon' ? <Icon size={props.size} className={css.toggleIndicator}>circle</Icon> : null}
```

Comment:
Hello @daniel-stoian-lgp 

I'm not sure that we should support the indicator with the `decoration` prop.

We support it in other skins except the `silicon` skin with `after` pseudo-element. If it is not our duty to use the `circle` icon, we could implement that part with css styles. Here is my test code to do it ( Only for silicon skin ) - https://github.com/enactjs/agate/commit/a66dd2c89a52846b940e2de413e5c5e32cdb006c.

Please let me know your thought.

CC: @Djspaceg , @seunghoh

Reply:
Hello @ybsung 
Your proposal looks good to me. I don't think also that we need to use a circle icon, but just a circle shaped indicator
Maybe the others have some comments. Otherwise, i will apply your proposal.

Reply:
I tried to reduce the css with &.huge {}, &.small {}, and &.smallest {}. But for now there is no idea. Let's revisit it after fixing above.